Plugin/Framework/Search/Request/AbstractHandler.php

```diff
@@ -281,15 +266,8 @@ private function getCmpProductIds(FacetInterface $facet, $pageNum, $limit)
                 $limit
             );
             return $res ? CategoryMerchandising::parseProductIds($res) : null;
-        } catch (MissingCookieException $e) {
-            $this->logger->debugWithSource(
-                $e->getMessage(),
-                [],
-                $this
-            );
-            return null;
         } catch (Exception $e) {
```

Comment:
Is there a need to catch this exception here? Look at the caller, it already has a try-catch block.